src/ui/control/attribution_control.js

```diff
@@ -66,11 +66,12 @@ class AttributionControl {
     }
 
     _updateEditLink() {
-        if (!this._editLink) this._editLink = this._container.querySelector('.mapboxgl-improve-map');
+        if (!this._editLink) this._editLink = this._container.querySelector('.mapbox-improve-map');
```

Comment:
correct @anandthakker - I will ticket out changing that css class at the API level separately